pkgs/applications/terminal-emulators/wayst/default.nix

```diff
@@ -83,6 +83,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A simple terminal emulator";
+    mainProgram = "wayst";
```

Comment:
```suggestion
```

thats the default.

Reply:
The unstable parsing is a bit broken right now.

Reply:
It's an issue with `builtins.parseDrvName` and the version format used for unstable releases.